future-apps/payroll/app/src/components/Table/Table.js

```diff
@@ -83,37 +189,61 @@ class Table extends React.Component {
       </TableRow>
     )
 
-    const body = filteredData.map(item => (
-      <TableRow key={`row-${item.id}`}>
-        {columns.map(column => {
-          const rawValue = column.value(item)
-          const formattedValue = rawValue != null
-            ? (column.formatter
+    const body = (
+      <React.Fragment>
+        {paginatedData.map(item => (
+          <TableRow key={`row-${item.id}`} style={{ height: `${tableRowHeight}px` }}>
+            {columns.map(column => {
+              const rawValue = column.value(item)
+              const formattedValue = rawValue != null
+              ? (column.formatter
                 ? column.formatter(rawValue)
                 : rawValue.toString()
-            )
-            : column.defaultValue
+              )
+              : column.defaultValue
 
-          return (
-            <TableCell
-              key={`row-${item.id}-${column.name}`}
-              {...column.cellProps}
-              children={column.render
-                ? column.render(formattedValue, rawValue)
-                : (<Text>{formattedValue}</Text>)
-              }
-            />
-          )
-        })}
-      </TableRow>
-    ))
+              return (
+                <TableCell
+                  key={`row-${item.id}-${column.name}`}
+                  {...column.cellProps}
+                  children={column.render
+                    ? column.render(formattedValue, rawValue)
+                    : (<Text>{formattedValue}</Text>)
+                  }
+                  />
+              )
+            })}
+          </TableRow>
+        ))}
+        {this.renderEmptyRows(emptyRows)}
+      </React.Fragment>
+    )
+
+    const footer = (
```

Comment:
We could have a styled-component for this

Reply:
Moved footer content to a styled component